.github/workflows/interface_auto.yml

```diff
@@ -0,0 +1,72 @@
+name: Deployment on DockerHub and Hugging Face
+
+# Trigger the workflow on push events to the 'uncleaned-branch' branch
+on:
+  push:
+    branches:
+      - uncleaned-branch
+
+jobs:
+  build-and-deploy:
+    # Run the job on the latest Ubuntu runner
+    runs-on: ubuntu-latest
+
+    steps:
+    # Step 1: Check out the latest code from the repository
+    - name: Checkout code
+      uses: actions/checkout@v3
```

Comment:
Use the latest version, 
```
uses: actions/checkout@v3
```

Reply:
I just checked the releases [here](https://github.com/actions/checkout/releases) and it appears the latest version is ```v4```, so I'll stick with it then.